src/backend/marsha/core/views.py

```diff
@@ -337,6 +338,7 @@ def _get_app_data(self, cache_key, lti=None, resource_id=None):
                     "flags": {
                         SENTRY: switch_is_active(SENTRY),
                         BBB: settings.BBB_ENABLED,
+                        LIVE_RAW: False,
```

Comment:
Like for `settings.BBB_ENABLED` you should add a new setting in `src/backend/marsha/core/settings.py` using `LIVE_RAW_ENABLED = values.BooleanValue(False)` and here use `settings.LIVE_RAW_ENABLED`. Doing this, the flag is easily configurable.